### PR TITLE
View cleanup fix

### PR DESCRIPTION
**Pull request by randomguy0815**

### CODE REVIEW

File: SKPhotoBrowser/SKAnimator.swift [hunk 1]
```diff
@@ -84,7 +84,9 @@ class SKAnimator: NSObject, SKPhotoBrowserAnimatorDelegate {
             let scrollView = browser.pageDisplayedAtIndex(browser.currentPageIndex) else {
```
Comment (fabb) [inline SKPhotoBrowser/SKAnimator.swift:85]: ouch. which of the guard conditions does fail and why?

File: SKPhotoBrowser/SKPhotoBrowserDelegate.swift [hunk 2]
```diff
@@ -63,15 +63,25 @@ import Foundation
...
-     Asks the delegate for the view for a certain photo. Needed to detemine the animation when presenting/closing the browser.
+     Asks the delegate for the view for a certain photo. Needed to detemine the animation when presenting the browser.
      
      - Parameter browser: reference to the calling SKPhotoBrowser
      - Parameter index: the index of the removed photo
      
      - Returns: the view to animate to
      */
-    @objc optional func viewForPhoto(_ browser: SKPhotoBrowser, index: Int) -> UIView?
+    @objc optional func presentationViewForPhoto(_ browser: SKPhotoBrowser, index: Int) -> UIView?
```
Comment (fabb) [inline SKPhotoBrowser/SKPhotoBrowserDelegate.swift:73]: why do we want to separate the delegate calls? so we can scroll to the image properly in ad detail?

**Review by fabb — APPROVED** [review]

File: SKPhotoBrowser/SKAnimator.swift (hunk 1 above)
Comment (randomguy0815) [reply SKPhotoBrowser/SKAnimator.swift:85]: The `browser.delegate?.viewForPhoto?(...` method if you don't implement that delegate method. Which I am not implementing on purpose  as another dismiss animation would be used then. And this animation does not fit very well with our image slider.

File: SKPhotoBrowser/SKPhotoBrowserDelegate.swift (hunk 2 above)
Comment (randomguy0815) [reply SKPhotoBrowser/SKPhotoBrowserDelegate.swift:73]: With just one method you are suppling just one view for the present and dismiss animation. We don't want the dismiss animation as it does not fit very well with our image slider. 
So, by splitting them we can use the presentation animation alone.

**Review by PatrickDotStar — APPROVED** [review]

**Pull request merged**